daffodil-lib/src/main/scala/org/apache/daffodil/util/SchemaUtils.scala

```diff
@@ -40,15 +40,13 @@ object SchemaUtils {
    * Constructs a DFDL schema more conveniently than having to specify all those xmlns attributes.
    */
 
-  def dfdlTestSchemaUnqualified(
-    includeImports: Seq[Node],
+  def dfdlTestSchemaUnqualified(includeImports: Seq[Node],
```

Comment:
It does not appear that I have a choice to tell Eclipse NOT to change whitespace like this. I would have to completely shut off auto-formatting.